app/pb_kits/playbook/pb_selectable_card/_selectable_card.html.erb

```diff
@@ -1,12 +1,12 @@
-<%= content_tag(:div,
+<%= content_tag(:label,
```

Comment:
Did you mean to leave this change? Can you change it back or provide some explanation?